client/src/components/TableLibrary/TableLibrary.js

```diff
@@ -1,63 +0,0 @@
-import React, { Component, Fragment } from 'react';
```

Comment:
Removed this file, as the component is now just contained within the `LibrarySearchPage` page/component